brave/src/main/java/brave/propagation/ExtraFieldPropagation.java

```diff
@@ -143,25 +150,20 @@ public FactoryBuilder addPrefixedFields(String prefix, Collection<String> fieldN
     }
 
     public Factory build() {
-      if (prefixedNames.isEmpty()) {
```

Comment:
most of the red here was just me trying to pay for the new code by removing old code